src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -209,7 +209,7 @@ class AgoraBusiness(permissionsDataSource: PermissionsDataSource) extends LazyLo
               }
 
               entityToInsertAction flatMap { entityToInsert =>
-                val userAccess = new AccessControl(username, AgoraPermissions(All))
+                val userAccess = AccessControl(username, AgoraPermissions(All))
```

Comment:
Removing usages of `new` ensures we follow requirements and use the proper `apply` method